framework/src/Volo.Abp.Auditing/Volo/Abp/Auditing/AuditingManager.cs

```diff
@@ -110,23 +110,23 @@ protected virtual void MergeEntityChanges(AuditLogInfo auditLog)
             }
         }
 
-        protected virtual async Task SaveAsync(DisposableSaveHandle saveHandle)
+        protected virtual async Task SaveAsync(DisposableSaveHandle saveHandle, bool throwExceptions = false)
```

Comment:
Instead of `bool throwExceptions = false`, `bool hideErrors = true` would be better.